scripts/zones/Batallia_Downs/npcs/qm_chocobo_tale.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- Area: Batallia Downs
+--  NPC: qm_chocobo_tale (???)
+--  Quest: A Chocobo's tale
+-----------------------------------
+local ID = require("scripts/zones/Batallia_Downs/IDs")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:messageSpecial(ID.text.FIND_NOTHING)
```

Comment:
If this is the NPC's default message there is a separate file this can go into now `DefaultActions.lua` for each zone. Should be able to find an example in another zone to replicate this. Would likely make it so you don't need to make a new lua for this NPC.